SR2022/2021-11-10-kickstart-details.md

```diff
@@ -0,0 +1,29 @@
+---
+to: Student Robotics 2022 registered teams
+subject: Kickstart Details
+---
+
+Hi teams
+
+Only a couple of days left until Saturday’s Kickstart event, so here are some more details on what to expect and what we need from you.
+
+## Kits
+
+If you don’t already have a kit, we will have contacted you regarding the delivery arrangements. We need you to sign a kit disclaimer form ASAP to ensure that the kit can arrive shortly after Kickstart.
+
+If you have a kit from a previous year, we will soon be sending you instructions on how to update the software and check that everything is present and correct.
+
+## Kickstart
+
+As you already know, [Kickstart](https://studentrobotics.org/events/sr2022/kickstart/) is at 1pm uk time this Saturday 13th November. The livestream will last for approximately an hour and we will explain this year’s game, the competition structure, and give an overview of the kit.
```

Comment:
```suggestion
As you already know, [Kickstart](https://studentrobotics.org/events/sr2022/kickstart/) is at 1pm UK time this Saturday 13th November. The livestream will last for approximately an hour and we will explain this year’s game, the competition structure, and give an overview of the kit.
```
(though I'd be tempted to say "GMT" rather than "UK time")

Reply:
Historically we've used "UK time" intentionally, as it doesn't need changing on / around DST boundaries, and is a bit less jargon for the majority of our teams to remember what timezone they're in.